index.html

```diff
@@ -2474,7 +2474,7 @@
                         <br>
                         <div class="tg-btnarea">
                         <a class="tg-btn" href="http://bit.ly/sponsorshippyconindia2017">Download Prospectus</a>
-                        <a class="tg-btn" href="mailto:sponsorship@in.pycon.org">Become A Sponser</a>
+                        <a class="tg-btn" href="mailto:sponsorship@in.pycon.org">Become A Sponsor</a>
```

Comment:
Please look at `README.md`. You have to make changes in template file and re-generate HTML